spec/services/export_activity_to_csv_spec.rb

```diff
@@ -9,6 +9,7 @@
     it "creates a CSV line representation of the Activity" do
       activity_presenter = ActivityPresenter.new(project)
       result = ExportActivityToCsv.new(activity: project, report: report).call
+      next_four_quarter_totals = ExportActivityToCsv.new(activity: project, report: report).next_four_quarter_forecasts
```

Comment:
Is it worth splitting `result` into:

```
export_activity_to_csv = ExportActivityToCsv.new(...)
result = export_activity_to_csv.call
next_four_quarter_totals = result.next_four_quarter_forecasts
```

...so that `result` and `next_four_quarter_totals` comes from the same instance?